### DISCUSSION TITLE
The first request is always performed even if it was cached before.

### DISCUSSION THREAD

**Discussion by VanTigranyan** [arthurfiorette/axios-cache-interceptor#915] [Q&A]
Hi. I am using Indexed DB storage for caching. In my test app, I perform the same request 3 times. I see that when the first request is made, the cache is saved, and the next subsequent requests receive the cached response. Then, if I reload the page, the first request is performed again, even though the cache persisted and could be reused on the next page load. My question is: did I do something wrong, or is this an intended behavior? If it is intended, what's the purpose, and how can I tweak it to use the existing cache? Thanks in advance.

**Comment by arthurfiorette** [comment]
I can't help you without a reproducible example.

https://axios-cache-interceptor.js.org/guide/debugging
